Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -37,12 +39,34 @@ namespace
 
 } // anonymous namespace
 
+FPrintTimer::FPrintTimer(float InResetTime)
+	: ResetTime(InResetTime)
+	, Timer(ResetTime)
+{}
+
+void FPrintTimer::SetResetTimer(float InResetTime)
+{
+	ResetTime = InResetTime;
+	Timer = InResetTime;
+}
+
+void FPrintTimer::Tick(float DeltaSeconds)
```

Comment:
What if this was TickAndCheck and returned a bool? Is this a cleaner API?

Reply:
So it could be. At one point I was checking ShouldPrint in two different places. If we implemented TickAndCheck this would mean we would have to cache the bool somewhere and then use that if we needed to print in two separate locations. 

That said I am no longer printing in two different parts of the code using this timer, however I like having that option.  
Let me know what you think.